### ISSUE TITLE
TinyMCE i18n

### ISSUE THREAD

**Issue by emu42** [voidlabs/mosaico#310]
Hello,

I have another question regarding multilanguage support. I know that TinyMCE has been translated into many languages including German, but I don't know how to set up Mosaico's Tiny editor with the user's language. Is there a way to pass initialization parameters to Tiny through Mosaico's init method?

Thanks in advance!

Regards,
 Edmund

**Comment by bago** [comment]
Hi, have a look at those issues:
#233 
#194
#239
#247
#178

You can configure tinymce via "mosaico viewModel plugin" (https://github.com/voidlabs/mosaico/wiki/Mosaico-Plugins)

Please report back your solution if it works.

**Comment by emu42** [comment]
Thanks! I just got it to work.

I added a plugin like so:
```
var l10nPlugin = function(vm) {
	  ko.bindingHandlers.wysiwyg.fullOptions['language'] = 'de';
}

```
But then the language pack for TinyMCE was still missing so I also downloaded and saved those packs to bower_components/tinymce/langs. Then I added that directory to the bowercopy task to get the packs where I need them:
`
          'plugins': 'tinymce/plugins',
**          'langs': 'tinymce/langs',**
          'notoregular': 'webfont-notosans/regular',
`

And finally, it looks like I have a fully translated version of Mosaico running (Mosaico menus, template and TinyMCE). 

Thanks for your continued support, Bago!

BTW we also corrected some of the German translations along the way and I'd like to contribute those back to Mosaico. I'll sign up on transifex later today...

**Comment by Shwele** [comment]
Sorry to open this issue, but I am unable to add newly translated lang from Transifex to my Mosaico on CiviCRM.

I have added .json file on path `/files/civicrm/ext/uk.co.vedaconsulting.mosaico/packages/mosaico/dist/lang`       
  
Colleague of mine translated it to Serbian today so I've added it in correspondent folder mentioned above.

So the question is... where to enable language I want for Mosaico?
If I should be adding it as plugin, like emu42 proposed, in what file should I add that?

Thanks in advance.
